client/src/assets/ArrowIcon.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Icon from 'assets/Icon.jsx';
+
+/**
+ * A wedge arrow icon
+ */
+function ArrowIcon(props) {
+  return (
+    <Icon viewBox="0 0 20 20" {...props}>
+      <path fill="none" d="M11.611,10.049l-4.76-4.873c-0.303-0.31-0.297-0.804,0.012-1.105c0.309-0.304,0.803-0.293,1.105,0.012l5.306,5.433c0.304,0.31,0.296,0.805-0.012,1.105L7.83,15.928c-0.152,0.148-0.35,0.223-0.547,0.223c-0.203,0-0.406-0.08-0.559-0.236c-0.303-0.309-0.295-0.803,0.012-1.104L11.611,10.049z" />
```

Comment:
Checkout react/icons i added it for the stars and a few other things they have a lot of icons you can use with a easy import

Reply:
The thing is regardless of the source of an icon we still want to use it as an extension of a base class. This allows us to control the styling of _all_ icons on the page in one place. For example if we wanted to change the color of our icons, we'd only have to change it in `Icon` instead of hunting through all of our source code for icons.

Using inline SVG (i.e. using an SVG element, which is what Icon is, which contains the vector `path` elements that draw the icons) is better from a performance standpoint since it avoids the repeated I/O operations that static images would require. Although this may be what react-icons is doing since I see they use the SVG paths on their website. We only need a few icons though so I preferred to find and inline the ones I needed rather than adding a dependency. Using inline SVGs also allows for more nuanced control over the display of the icon. I couldn't find a way to meet the full spec of the star ratings without inline SVG. However, it was definitely a headache setting all of that up and react-icons streamlines the process considerably.